install/configuring_inventory_file.adoc

```diff
@@ -400,6 +400,9 @@ the default SDN. If enabling *flannel*, disable the default SDN with the
 |`openshift_sdn_vxlan_port`
 |This variable sets the `vxlan port` number for `cluster network`. Defaults to `4789`. See xref:../install_config/configuring_sdn.html#config-changing-vxlan-port-for-cluster-network[Changing the VXLAN PORT for the cluster network] for more information.
 
+|`openshift_node_sdn_mtu`
+|This variable specifies the MTU size to use for OpenShift SDN. The value must be `50` bytes less than the MTU of the primary network interface of the node. For example, if the primary network interface has an MTU of `1500`, this value will be `1450`. The MTU value is discovered automatically but can be overwritten if necessary.
```

Comment:
The "discovered automatically" part is a 4.x feature. In 3.x it defaults to 1450 unconditionally, so if your nodes don't have MTU 1500, you definitely need to override this.

This is mentioned in a few other other places in the docs already, but they all talk about editing the node config directly, not using ansible, so maybe there should be some updating/xreffing. eg, `install_config/configuring_sdn.adoc`, `day_two_guide/topics/verifying_mtu.adoc`, `scaling_performance/network_optimization.adoc`, `admin_guide/sdn_troubleshooting.adoc`

Reply:
@danwinship, ah, cool, glad I asked!

Reply:
@danwinship, I've omitted the automatic discovery sentence. OCP 3.x is old enough that I'm hesitant to rework any of the other content that discusses modifying the MTU on the node to incorporate this.